rosidl_default_generators/package.xml

```diff
@@ -25,11 +25,6 @@
   <buildtool_export_depend>rosidl_typesupport_introspection_c</buildtool_export_depend>
```

Comment:
I haven't done anything with them yet because they don't affect the vendor typesupport isolation but I think the same over-specification this PR changes for vendor typesupport packages applies to the introspection packages as well. Should I pull them out as well or leave that for a potential follow up PR?

Reply:
I would lean towards not changing it now if we don't have to.